src/Microsoft.AspNet.Mvc.Core/ModelBinding/MutableObjectModelBinder.cs

```diff
@@ -430,8 +436,20 @@ internal void ProcessDto(ModelBindingContext bindingContext, ComplexModelDto dto
                         out requiredValidator);
 
                     SetProperty(bindingContext, modelExplorer, propertyMetadata, dtoResult, requiredValidator);
+
+                    var dtoValidationNode = dtoResult.ValidationNode;
+                    if (dtoValidationNode == null)
+                    {
+                        // Make sure that irrespective of if the properties of the model were bound with a value,
+                        // create a validation node so that these get validated.
+                        dtoValidationNode = new ModelValidationNode(dtoResult.Key, entry.Key, dtoResult.Model);
+                    }
+
+                    validationNode.ChildNodes.Add(dtoValidationNode);
```

Comment:
Well I hate even the existence of a `ComplexDTOModelBinder` now the code to handle a validation node is in 3 places. But :ok: if that makes you two happy.